docs/keycodes_basic.md

```diff
@@ -191,7 +191,7 @@ The basic set of keycodes are based on the [HID Keyboard/Keypad Usage Page (0x07
 
 These keycodes are not part of the Keyboard/Keypad usage page. The `SYSTEM_` keycodes are found in the Generic Desktop page, and the rest are located in the Consumer page.
 
-Windows and macOS use different keycodes for "next track" and "previous track". Make sure you choose the keycode that corresponds to your OS.
+Note that some of these keycodes may be OS-dependent. For example, MacOS supports the keycodes `KC_MEDIA_FAST_FORWARD` and `KC_MEDIA_REWIND` in addition to `KC_MEDIA_NEXT_TRACK` and `KC_MEDIA_PREV_TRACK`: those extra keycodes skip within the current track when held, but skip the entire track like the latter keycodes when tapped.
```

Comment:
I still think we could do a little wordsmithing here. Also enclosed in an info box to draw attention to it.

```suggestion
?> Some of these keycodes may behave differently depending on the OS. For example, on macOS, the keycodes `KC_MEDIA_FAST_FORWARD`, `KC_MEDIA_REWIND`, `KC_MEDIA_NEXT_TRACK` and `KC_MEDIA_PREV_TRACK` skip within the current track when held, but skip the entire track when tapped.
```

Reply:
Maybe, but you've changed the meaning: I believe that only the additional keycodes skip within the current track, but your edited version claims that the others do that too.

Reply:
You might be right about that, in which case you should feel free to accept your own edit and merge it this is otherwise helpful.

Reply:
Yep, I can confirm both sets behave exactly the same, at least on Catalina.

Reply:
Great, done - thanks!